cabal.project.newbuild

```diff
@@ -19,11 +19,11 @@ packages:
 optional-packages:
   submodules/arm-asl-parser
   submodules/asl-translator
+  submodules/bv-sized
```

Comment:
Do we want the submodule or the hackage version?

Reply:
Hackage version. Thanks